recording-oracle/src/modules/campaigns/campaign.entity.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  BeforeInsert,
+  BeforeUpdate,
+  Column,
+  Entity,
+  Index,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+import { DATABASE_SCHEMA_NAME } from '@/common/constants';
+
+import { CampaignStatus } from './types';
+
+@Entity({ schema: DATABASE_SCHEMA_NAME, name: 'campaigns' })
+@Index(['chainId', 'address'], { unique: true })
+export class CampaignEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column('int')
+  chainId: number;
+
+  @Column('varchar', { length: 42, unique: true })
```

Comment:
```suggestion
  @Column('varchar', { length: 42 })
```

Should we have only `['chainId', 'address']` unique index? If we have one just for address, then there will be no opportunity to have same address for different chains (I'm not sure it's possible though)